### ISSUE TITLE
Display a coloured message in chat when someone gets a +5 or more

### ISSUE THREAD

**Issue by bunniewormy** [transhumandesign/kag-base#628]
**Describe the feature**

Title, makes these kills more satisfying, something like 'Homek got a Pentakill!' in a special colour in chat.

**Comment by benjamingorman** [comment]
Where do you think the message should appear? As part of the kill feed? Or above the player? Or both?

**Comment by bunniewormy** [comment]
did you read the title?

**Comment by benjamingorman** [comment]
Of course. The chat box is not necessarily the best place though. It's perhaps not as visible as text in game - or as obviously linked to the player in question.
